networkx/addons/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
I think there are many possibilities arising regarding this but only one would be correct. I'll try them and see which one works.